.teamcity/src/main/kotlin/util/RerunFlakyTest.kt

```diff
@@ -33,9 +34,9 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
 import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
 import jetbrains.buildServer.configs.kotlin.v2019_2.ParameterDisplay
 
-class RerunFlakyTest(os: Os) : BuildType({
-    val id = "Util_RerunFlakyTest${os.asName()}"
-    name = "Rerun Flaky Test - ${os.asName()}"
+class RerunFlakyTest(os: Os, arch: Arch = Arch.AMD64) : BuildType({
+    val id = "Util_RerunFlakyTest${os.asName()}${arch.asName()}"
```

Comment:
So we are going to lose the history here, right? I don't think that is a problem, just pointing it out.

Reply:
Right, I also don't think that's a big deal.